src/components/Footer.tsx

```diff
@@ -0,0 +1,31 @@
+import Location from './Location';
+
+import { FooterDiv, ContactDiv } from '../styles/components';
+
+export default function Logo() {
```

Comment:
change the function name, Logo gets reused across multiple pages rn. generally good practice is to name it after the page or component i.e. here it would be export default function Footer()